CRM/Financial/BAO/PaymentProcessor.php

```diff
@@ -120,6 +120,26 @@ public static function getCreditCards($paymentProcessorID = NULL) {
     return [];
   }
 
+  /**
+   * Get options for a given contribution field.
+   *
+   * @param string $fieldName
+   * @param string $context see CRM_Core_DAO::buildOptionsContext.
+   * @param array $props whatever is known about this dao object.
+   *
+   * @return array|bool
+   * @see CRM_Core_DAO::buildOptions
+   *
+   */
+  public static function buildOptions($fieldName, $context = NULL, $props = []) {
+    $params = [];
+    if ($fieldName === 'financial_account_id') {
```

Comment:
@seamuslee001 FYi this is to help on apiv3 but will also help on your apiv4 version